package.json

```diff
@@ -1,83 +1,32 @@
+    "globby": "^6.1.0",
+    "lerna": "^2.11.0"
   },
-  "bugs": {
-    "url": "https://github.com/okta/okta-auth-js/issues"
+  "engines": {
+    "yarn": "^1.7.0"
   },
   "scripts": {
-    "lint": "eslint .",
-    "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml .",
+    "clean": "yarn reset-node-modules",
+    "reset-node-modules": "lerna exec rimraf node_modules && rimraf node_modules",
+    "lint": "yarn workspaces run lint",
+    "publish": "lerna publish --skip-npm",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:e2e": "yarn --cwd test/e2e start",
-    "test:unit": "yarn test:karma && yarn test:browser && yarn test:server",
-    "test:karma": "karma start --single-run",
-    "test:browser": "jest --config ./jest.browser.js",
-    "test:server": "jest --config ./jest.server.js",
+    "test:e2e": "yarn workspace @test/e2e start",
+    "test:unit": "yarn workspace @okta/okta-auth-js test",
     "test:report": "yarn test --ci --silent || true",
-    "build": "node ./writeConfig.js && webpack --config webpack.config.js",
-    "prepare": "yarn build",
-    "start": "yarn --cwd test/app start --open"
-  },
-  "author": "Okta",
-  "keywords": [
-    "Okta",
-    "authentication",
-    "auth",
-    "login"
-  ],
-  "dependencies": {
-    "Base64": "0.3.0",
-    "cross-fetch": "^3.0.0",
-    "js-cookie": "2.2.0",
-    "node-cache": "^4.2.0",
-    "q": "1.4.1",
-    "reqwest": "2.0.5",
-    "tiny-emitter": "1.1.0",
-    "xhr2": "0.1.3"
-  },
-  "devDependencies": {
-    "eslint": "5.6.1",
-    "istanbul-instrumenter-loader": "^3.0.1",
-    "jasmine-ajax": "^4.0.0",
-    "jest": "^23.6.0",
-    "jest-junit": "^5.0.0",
-    "jquery": "3.3.1",
-    "json-loader": "0.5.4",
-    "karma": "^4.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage-istanbul-reporter": "^2.0.5",
-    "karma-jasmine": "^2.0.1",
-    "karma-jquery": "^0.2.3",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^3.0.5",
-    "lodash": "4.17.11",
-    "promise.allsettled": "^1.0.1",
-    "promise.prototype.finally": "^3.1.1",
-    "webpack": "^3.0.0"
-  },
-  "okta-auth-js": {
-    "STATE_TOKEN_KEY_NAME": "oktaStateToken",
-    "DEFAULT_POLLING_DELAY": 500,
-    "DEFAULT_MAX_CLOCK_SKEW": 300,
-    "DEFAULT_CACHE_DURATION": 86400,
-    "FRAME_ID": "okta-oauth-helper-frame",
-    "REDIRECT_OAUTH_PARAMS_COOKIE_NAME": "okta-oauth-redirect-params",
-    "REDIRECT_STATE_COOKIE_NAME": "okta-oauth-state",
-    "REDIRECT_NONCE_COOKIE_NAME": "okta-oauth-nonce",
-    "TOKEN_STORAGE_NAME": "okta-token-storage",
-    "CACHE_STORAGE_NAME": "okta-cache-storage",
-    "PKCE_STORAGE_NAME": "okta-pkce-storage"
+    "prepare": "yarn workspace @okta/okta-auth-js build",
+    "start": "yarn workspace @test/app start --open"
   },
-  "jest-junit": {
-    "output": "./build2/reports/unit/junit-result.xml"
-  }
+  "workspaces": [
+    "packages/*",
+    "packages/@okta/*",
+    "packages/@test/*"
```

Comment:
I'm not a fan of creating our own fake namespace - is this a conventional practice?

Reply:
As we discussed, I will remove the `@` prefix for test